teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './main'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
```

Comment:
```suggestion
 def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
```

- [ ] Kindly add `age, name, parent_permission` as parameters in your constructor